README.md

```diff
@@ -1,52 +1,64 @@
-# Shazam-3-scrum-unleashed
-
-Project 3 - FIT3170 - 2024
-
-Team Members:
-
-    - Aaron Abbott (aabb0010@student.monash.edu) Repo Admin
-    - Patrick (pedw0004@student.monash.edu)
-    - James Chea (jche0346@student.monash.edu)
-    - Levi Kogan (lkog0002@student.monash.edu)
-    - Michael (mwan0129@student.monash.edu)
-    - Suryadeep Singh (ssin0061@student.monash.edu)
-    - Alexander Chan (acha0145@student.monash.edu)
-    - Anand Vannalath (avan0055@student.monash.edu)
-    - Victor Huang (vhua006@student.monash.edu)
-    - Sandy (ssen0027@student.monash.edu)
-    - Snekith (skar0053@student.monash.edu)
-    - Jovan Vales (jval0006student.monash.edu)
-    - Fathy Abdelshahid (fabd0009@student.monash.edu)
-    - Rory Tobin-Underwood (rtob0002@student.monash.edu)
-
-# Repo Structure
+# Shazam-3-Scrum-Unleashed
+
+### Project 3 - FIT3170 - 2024
+
+## Team Members
+- Aaron Abbott (aabb0010@student.monash.edu)
+- Patrick (pedw0004@student.monash.edu)
+- James Chea (jche0346@student.monash.edu)
+- Levi Kogan (lkog0002@student.monash.edu)
+- Michael (mwan0129@student.monash.edu)
+- Suryadeep Singh (ssin0061@student.monash.edu)
+- Alexander Chan (acha0145@student.monash.edu)
+- Anand Vannalath (avan0055@student.monash.edu)
+- Victor Huang (vhua006@student.monash.edu)
+- Sandy (ssen0027@student.monash.edu)
+- Snekith (skar0053@student.monash.edu)
+- Jovan Vales (jval0006@student.monash.edu)
+- Fathy Abdelshahid (fabd0009@student.monash.edu)
+- Rory Tobin-Underwood (rtob0002@student.monash.edu)
+
+## Developed Application
+The developed application is currently available [here](https://3170.fit/) to be enjoyed.
```

Comment:
Can't guarantee that I will continue hosting the backend after this unit is over though. So not sure if we want to put this here for handover.

Reply:
We can change it when that happens